apm-agent-core/src/main/java/co/elastic/apm/agent/bci/ElasticApmAgent.java

```diff
@@ -257,8 +259,19 @@ private static synchronized void initInstrumentation(final ElasticApmTracer trac
         Runtime.getRuntime().addShutdownHook(new Thread(ThreadUtils.addElasticApmThreadPrefix("init-instrumentation-shutdown-hook")) {
             @Override
             public void run() {
+                if (tracer.getConfig(CoreConfiguration.class).logUsedInstrumentationGroupsOnExit()) {
```

Comment:
Instead of a new config option, maybe just log on debug?

Reply:
My motivation is to enable this function without having to enable debug logs. Enabling debug logs produces many too many logs and negatively impacts performance.

What do you think about removing the config option and keeping the log level at info? As it is only logged when the application exists, it shouldn't negatively impact the performance.

Reply:
Done